tests/unit/amo/components/TestSearchFilters.js

```diff
@@ -25,18 +25,19 @@ import {
 describe(__filename, () => {
   let fakeHistory;
 
-  function render({
-    filters = {},
-    pathname = '/en-US/android/search/',
-    ...props
-  } = {}) {
+  function render({ filters = {}, pathname = '/search/', ...props } = {}) {
```

Comment:
Is there a reason for removing the clientApp and lang from pathname here?

Reply:
Yes, this prop name is a bit misleading: it is a developer-defined prop called pathname, not the pathname from the routing layer. It should be called something like "the last part of the pathname".

It is supplied via the `Search` component, which defines the default value:  https://github.com/mozilla/addons-frontend/blob/82c44dad7d818dadc60d17e9b93e88088edd4779/src/amo/components/Search/index.js#L69